packages/backend-core/src/middleware/contentSecurityPolicy.ts

```diff
@@ -89,6 +89,8 @@ const CSP_DIRECTIVES = {
   "worker-src": ["blob:", "'self'"],
 }
 
+const CSPDomainRegex = /^[A-Za-z0-9-*:/.]+$/
```

Comment:
Is this definitely good enough? If someone enters "........" which this regex will consider valid, is that going to cause problems? Does Node not have some robust way to validate domains?

Reply:
Yea if you enter something like that it'll log a different error saying it's an invalid source and it will be ignored, which I think is harmless. As long as it doesn't brick entirely, which happens if illegal characters are entered. Validating the domains would certainly be more robust but I don't think it's necessarily required here, since if someone wants to enter harmless garbage then they can.

Reply:
I'm iffy about it, I think we could try and be more helpful if the user makes a mistake, but in the spirit of getting this fix out it's fine.